SettingsList.py

```diff
@@ -4735,6 +4735,13 @@ def get_settings_from_tab(tab_name):
                     yield setting
             return
 
+def get_all_settings():
```

Comment:
What's the benefit of iterating over the setting_map like this, as opposed to getting the names out of `setting_infos`? (Is ordering important?)

Reply:
There is none :p Thanks for pointing that out, will change/remove the new function